packages/neo-one-node-core/src/Block.ts

```diff
@@ -44,11 +45,11 @@ export interface BlockAdd {
   readonly version?: number;
   readonly previousHash: UInt256;
   readonly merkleRoot?: UInt256;
-  readonly timestamp: number;
+  readonly timestamp: BN;
```

Comment:
Are we changing this for general integer safety?  The change to BN from number is causing problems elsewhere when the timestamp needs to be added to another number that isn't a BN. It's likely not a big deal to deal convert a number to BN when needing to use it together with the timestamp, but I want to understand why we might need timestamp to be a BN